services/httpd/handler.go

```diff
@@ -134,7 +133,7 @@ func (h *Handler) SetRoutes(routes []route) {
 		if r.gzipped {
 			handler = gzipFilter(handler)
 		}
-		handler = versionHeader(handler, h.version)
+		handler = versionHeader(handler, h)
```

Comment:
I typically prefer to only send what is needed to the receiving method, which in this case is just the version string.  It makes the receiving function easy to understand what is actually needed.